modules/src/ics02_client/client_def.rs

```diff
@@ -282,6 +284,23 @@ pub enum AnyConsensusState {
 }
 
 impl AnyConsensusState {
+    pub fn latest_timestamp(&self) -> u64 {
```

Comment:
Two ideas:

1. Would make sense to rename this into `timestamp` as there is no earliest or latest timestamp in a single consensus state.

2. We should try to handle the panic in converting the integer here. One solution: use the value `0` as an invalid timestamp which we return if conversion failed; to catch the error in converting integer -> uint we can use try_from. Example: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=56d9637ca51dafadc0f4bbf4b18f6227

Reply:
Done&Done.